static/app/views/replays/detail/network/index.tsx

```diff
@@ -246,16 +313,64 @@ function NetworkList({replayRecord, networkSpans}: Props) {
           query={searchTerm}
         />
       </NetworkFilters>
-      <StyledPanelTable
-        columns={columns.length}
-        isEmpty={networkData.length === 0}
-        emptyMessage={t('No related network requests found.')}
-        headers={columns}
-        disablePadding
-        stickyHeaders
-      >
-        {networkData.map(renderTableRow)}
-      </StyledPanelTable>
+
+      <NetworkTable ref={networkTableRef}>
+        <AutoSizer>
+          {({width, height}) => (
+            <MultiGrid
+              ref={multiGridRef}
+              columnCount={columns.length}
+              columnWidth={({index}) => {
+                if (index === 0) {
+                  // For the status code column, we want to set a fixed width
+                  return Math.max(80, cache.columnWidth({index}));
+                }
+
+                // If the column is the path column, we want to take up the rest of the space
+                if (index === 1) {
```

Comment:
This is to make the second column to take the space remaining of the container, working as if we had `1fr` for its width